.evergreen/config.yml

```diff
@@ -450,6 +464,15 @@ functions:
         working_dir: src/go.mongodb.org/mongo-driver
         script: |
           ${PREPARE_SHELL}
+
+          # Set temp credentials for AWS.
```

Comment:
Can you explain why the `PYTHON3_BINARY` check is not needed here but is needed in `run-tests`?

Reply:
Yep! The `run-versioned-api-test` command is only on latest servers, so the issue of a non-existent `PYTHON3_BINARY` env variable on 2.6 and 3.0 servers is irrelevant.